ecomerce-app/src/context/Prodctcard-context.js

```diff
@@ -0,0 +1,33 @@
+// import {  } from "react";
+import { useAxios } from "../Utils/useAxios"
+import { useContext,createContext, useState, useEffect } from "react"
+
+const Productcontext = createContext();
+
+const Productprovider = ({ children }) => {
```

Comment:
by convention we write providers like this: ProductProvider that is p of provider in capital.